lib/integer.gd

```diff
@@ -799,7 +799,7 @@ DeclareGlobalFunction( "NextPrimeInt" );
 ##
 ##  <Description>
 ##  returns <M><A>r</A>^{<A>e</A>} \pmod{<A>m</A>}</M> for integers <A>r</A>,
-##  <A>e</A> and <A>m</A> (<M><A>e</A> \geq 0</M>).
+##  <A>e</A> and <A>m</A>.
 ##  <P/>
```

Comment:
But e can (still) be 0. And negative values were supported before, just not documented. 
Documenting this more explicitly is fine by me, but I'd prefer to not do this as part of this PR (which has been in the works for very long by now)...